EMMO/README.md

```diff
@@ -0,0 +1,74 @@
+[https://raw.githubusercontent.com/emmo-repo/emmo-repo.github.io/master/latest/emmo.ttl](https://raw.githubusercontent.com/emmo-repo/emmo-repo.github.io/master/versions/1.0.0-beta5/emmo.ttl)
+
+2. `https://w3id.org/emmo/`
+   - `emmo.ttl` file in the root of the master branch: https://raw.githubusercontent.com/emmo-repo/EMMO/master/emmo.ttl
+
+3. `https://w3id.org/emmo/{VERSION}`
+   - If the user is accessing this from a browser: redirect to documentation of given version corresponding IRI (`https://w3id.org/emmo/{VERSION}#{NAME}`).
+
+     Otherwise, redirect to squashed ontology on GitHub Pages:
+https://raw.githubusercontent.com/emmo-repo/emmo-repo.github.io/master/versions/{VERSION}/emmo.ttl
+
+
+4. `https://w3id.org/emmo/{VERSION}/`
+   - `emmo.ttl` file in the root of branch for the given version: https://raw.githubusercontent.com/emmo-repo/EMMO/{VERSION}/emmo.ttl
+
+5. `https://w3id.org/emmo/{PATH}/{MODULE}`
+   - Turtle file for given module of EMMO:
+   https://raw.githubusercontent.com/emmo-repo/EMMO/{PATH}/{MODULE}.ttl
+
+6. `https://w3id.org/emmo/{VERSION}/{PATH}/{MODULE}`
+   - Turtle file for given version and module of EMMO: https://raw.githubusercontent.com/emmo-repo/EMMO/{VERSION}/{PATH}/{MODULE}.ttl
+
+7. `https://w3id.org/emmo/{DOMAIN}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of corresponding IRI (`https://w3id.org/emmo/{DOMAIN}#{NAME}`).
+     Ex: http://emmo.info/electrochemistry#electrochemistry_214d925c_76c4_4f69_9afc_056a1ea82fc6
+
+   - Otherwise, redirect to squashed ontology on GitHub Pages:
+     https://emmo-repo.github.io/domain-electrochemistry/latest/{DOMAIN}.ttl
+
+8. `https://w3id.org/emmo/{DOMAIN}/`
+   - `{DOMAIN}.ttl` file in the root of the master branch.
+
+9. `https://w3id.org/emmo/{DOMAIN}/{VERSION}`
+   - If the user is accessing this from a browser: redirect to github pages documentation of corresponding IRI for given version (`https://w3id.org/emmo/{DOMAIN}/{VERSION}#{NAME}`).
+
+   - Otherwise, redirect to squashed ontology on GitHub Pages:
+     https://emmo-repo.github.io/domain-electrochemistry/versions/{VERSION}/{DOMAIN}.ttl
+
+10. `https://w3id.org/emmo/{DOMAIN}/{VERSION}/`
+   - `{DOMAIN}.ttl` file in the root of the branch `{VERSION}`
+
+   - `{DOMAIN}.ttl` file in the root of branch for the given version: https://raw.githubusercontent.com/emmo-repo/{DOMAIN}/{VERSION}/{DOMAIN}.ttl
+
+11. `https://w3id.org/emmo/{DOMAIN}/{PATH}/{MODULE}`
+    - `{PATH}/{MODULE}.ttl` file in master branch:
+https://raw.githubusercontent.com/emmo-repo/{DOMAIN}/master/{PATH}/{MODULE}.ttl
+
+12. `https://w3id.org/emmo/{DOMAIN}/{VERSION}/{PATH}/{MODULE}`
+    - `{PATH}/{MODULE}.ttl` file in branch `{VERSION}` (`{PATH}` may be empty if the module .ttl file is in the repository root):
+https://raw.githubusercontent.com/emmo-repo/{DOMAIN}/{VERSION}/{PATH}/{MODULE}.ttl
+
+
+Meaning of placeholders:
+`{NAME}`: Name part of an IRI, i.e. what follows after the hash sign.
```

Comment:
You should put these in a markdown list or add trailing specs so it all renders properly.